web/templates/single_message.hbs

```diff
@@ -5,9 +5,13 @@
     {{#if want_date_divider}}
     <div class="date_row no-select">{{{date_divider_html}}}</div>
     {{/if}}
-    <div class="messagebox">
-        <div class="messagebox-content">
-            {{> message_body}}
+    <div class="messagebox-background">
+        <div class="messagebox-hover-overlay">
+            <div class="messagebox">
+                <div class="messagebox-content">
+                    {{> message_body}}
+                </div>
+            </div>
```

Comment:
This change belongs in the later, hover commit, right?

Reply:
It was easier to keep the template setup for hover in the first commit, but if you prefer that be moved to the second commit I can do that.

Reply:
(easier as in no intermediate code changes)

Reply:
But I can do it more properly now and split it out :)